SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/OperationPreferredDateCalculator.swift

```diff
@@ -83,7 +83,7 @@ struct OperationPreferredDateCalculator {
             return date.now.addingTimeInterval(calculateNextRunDateOnError(schedulingConfig: schedulingConfig, historyEvents: historyEvents))
         case .optOutStarted, .scanStarted, .noMatchFound:
             return currentPreferredRunDate
-        case .optOutConfirmed, .matchRemovedByUser:
+        case .optOutConfirmed, .matchRemovedByUser, .optOutSubmittedAndAwaitingEmailConfirmation:
```

Comment:
This is meant as placeholder for the actual changes to come in a PR up stack, so we don’t need to calculate the actual date for this case yet.